package.json

```diff
@@ -1,15 +1,16 @@
 {
   "scripts": {
     "start": "pnpm run dev",
-    "dev": "NODE_ENV=development esmo ./server/index.ts",
+    "dev": "SET NODE_ENV=development & esmo ./server/index.ts",
     "prod": "pnpm run build && pnpm run prod:serve",
     "// Build for production": "",
     "build": "pnpm run build:clean && pnpm run build:vite && npm run build:prerender",
     "build:clean": "rimraf dist/",
     "build:vite": "vite build && vite build --ssr",
     "build:prerender": "vite-plugin-ssr prerender",
     "// Static serve for production": "",
-    "prod:serve": "serve dist/client/ --listen 3000"
+    "prod:serve": "serve dist/client/ --listen 3000",
+    "prisma": "npx prisma migrate dev --name init"
```

Comment:
Planet scale prefers `npx prisma db push` to `npx prisma migrate`

https://docs.planetscale.com/tutorials/prisma-quickstart